free-programming-books-tr.md

```diff
@@ -83,6 +84,11 @@
 * [iOS Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/ios)
 
 
+### Java
+
+* [Java ile 24 Kahve Molası](https://www.buraksenyurt.com/pics/2014%2f2%2fJavaile24KahveMolasi.docx) - [Burak Selim Şenyurt]
```

Comment:
is there something safer than a word document?

Reply:
It is difficult for me to find an alternative resource because I don't read Turkish.
Should I keep or remove the link?

Reply:
remove